nim.cfg

```diff
@@ -0,0 +1,3 @@
+@if nimHasWarningObservableStores:
+  --warning[ObservableStores]: off
```

Comment:
https://forum.nim-lang.org/t/6442#39738

> There is a new warning about "Observable Stores". I have yet to see a case where this warning indicates a real problem and so your config should probably disable it. Code to do that is:

lol? Why did you implement this warning and enable it by default then @Araq?

I'd honestly rather not pollute Nimble's nim.cfg with stuff like this if the compiler appears to be to blame.

Reply:
Yeah, but for now it occurs soooo many times. It dominates the output.

Reply:
I'd rather wait for the warning to be disabled by default which it sounds like it should be.